forge-gui/res/cardsfolder/upcoming/tyrannical_pitlord.txt

```diff
@@ -0,0 +1,12 @@
+Name:Tyrannical Pitlord
+ManaCost:4 B B
+Types:Creature Demon
+PT:6/6
+K:Flying
+K:Trample
+K:ETBReplacement:Other:DBChoose
+SVar:DBChoose:DB$ ChooseCard | Choices$ Creature.Other+YouCtrl | RememberChosen$ True | TgtPrompt$ Select another target creature you control | Description$ As CARDNAME enters the battlefield, choose another creature you control.
```

Comment:
I'd say do this all with Chosen rather than Remembered – either way, I think you want a cleanup after the changeszone trigger